config/routes.rb

```diff
@@ -2389,8 +2389,10 @@
         index
         show
         show_list
+        new
       ],
       :post => %w[
+        button
```

Comment:
do we need this route?

Reply:
@h-kataria you are correct, we dont need it since we removed the button functionality from the controller. I removed it